buildSrc/build.gradle

```diff
@@ -0,0 +1,5 @@
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(8)
```

Comment:
Did you verify this works after purging all JDK 8 installations from your computer? I think this will fail to provision since Adoptium (the default) does not distribute compatible binaries for macOS aarch64. It's why in the BVS repo we set it to Azul. 

https://github.com/gradle/gradle-enterprise-build-validation-scripts/blob/d2f69dc611450fd4e0345f44626a0dce13889a52/buildSrc/build.gradle.kts#L8

Reply:
Ah good insight, let me test that case

Reply:
This is not the behavior I'm seeing. This is after clearing out downloaded toolchains and disabling auto-discovery (` -Porg.gradle.java.installations.auto-detect=false`). You can see that it provisions Azul: https://ge.solutions-team.gradle.com/s/mo36e4mvi4sjw/timeline?details=ozqpw2bwus4gq

Reply:
This appears to be the behavior of Gradle 7, when it auto-provisioned toolchains: https://ge.solutions-team.gradle.com/s/v2ts5cghaiyhy/failure#1

Foojay seems to take into account the architecture: https://github.com/gradle/foojay-toolchains/blob/312cabcaddf8352e8d62e5db6e765b7c40f88db5/foojay-resolver/src/main/kotlin/org/gradle/toolchains/foojay/packages.kt#L19

Reply:
Ah, nice! I didn't realize the plugin did that. We could probably also remove the explicit vendor in the BVS repo too. 👍